index.html

```diff
@@ -10353,46 +10353,6 @@ <h2 id="Performance">
           more work than is possible in real-time given the CPU's speed.
         </p>
       </section>
-      <section>
-        <h3 id="JavaScriptPerformance">
-          JavaScript Issues with Real-Time Processing and Synthesis:
-        </h3>While processing audio in JavaScript, it is extremely challenging
-        to get reliable, glitch-free audio while achieving a reasonably
-        low-latency, especially under heavy processor load.
-        <ul>
-          <li>JavaScript is very much slower than heavily optimized C++ code
-          and is not able to take advantage of SSE optimizations and
-          multi-threading which is critical for getting good performance on
-          today's processors. Optimized native code can be on the order of
-          twenty times faster for processing FFTs as compared with JavaScript.
-          It is not efficient enough for heavy-duty processing of audio such as
-          convolution and 3D spatialization of large numbers of audio sources.
-          </li>
-          <li>setInterval() and XHR handling will steal time from the audio
-          processing. In a reasonably complex game, some JavaScript resources
-          will be needed for game physics and graphics. This creates challenges
-          because audio rendering is deadline driven (to avoid glitches and get
-          low enough latency).
-          </li>
-          <li>JavaScript does not run in a real-time processing thread and thus
-          can be pre-empted by many other threads running on the system.
-          </li>
-          <li>Garbage Collection (and autorelease pools on Mac OS X) can cause
-          unpredictable delay on a JavaScript thread.
```

Comment:
It's perfectly possible to write js that does not GC. In fact, that's recommended. emscripten outputs js that does not gc.